src/pages/DailyGardening/index.tsx

```diff
@@ -164,15 +167,23 @@ export const DailyGardening = () => {
 
   const classes = useStyles();
   return (
-    <motion.div
-      initial={{ opacity: 0 }}
-      animate={{ opacity: 1 }}
-      transition={{ duration: 0.3 }}
-      exit={{ opacity: 0 }}
-    >
-      <div className={styles.gardenParentContainer}>
-        <h1>Daily Gardening</h1>
-        <LoadingWrapper isLoading={!gardenDataApi.isLoaded}>
+    <>
+      <Head title="Daily Gardening" />
+      <motion.div
```

Comment:
wrap with fragment and add Head component
and added common component for layout consistency